src/main/java/uk/gov/hmcts/reform/judicialapi/controller/JrdUsersController.java

```diff
@@ -0,0 +1,90 @@
+package uk.gov.hmcts.reform.judicialapi.controller;
+
+import com.nimbusds.oauth2.sdk.util.CollectionUtils;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+import io.swagger.annotations.Authorization;
+import lombok.AllArgsConstructor;
+import lombok.NoArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.access.annotation.Secured;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.hmcts.reform.judicialapi.controller.advice.InvalidRequestException;
+import uk.gov.hmcts.reform.judicialapi.controller.request.UserRequest;
+import uk.gov.hmcts.reform.judicialapi.controller.response.OrmResponse;
+import uk.gov.hmcts.reform.judicialapi.service.JudicialUserService;
+
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+
+@RequestMapping(
+    path = "/refdata/judicial/users"
+)
+@RestController
+@Slf4j
+@NoArgsConstructor
+@AllArgsConstructor
+public class JrdUsersController {
+
+
+    @Autowired
+    JudicialUserService judicialUserService;
+
+    @ApiOperation(
+            value = "This API returns judicial user profiles with their appointments and authorisations",
+            authorizations = {
+                 @Authorization(value = "ServiceAuthorization"),
+                 @Authorization(value = "Authorization")
+            }
```

Comment:
Add note section which mentions sidam role which is used to  call this api , similar has been done on PRD swagger